test/src/commonMain/kotlin/ManualClock.kt

```diff
@@ -0,0 +1,32 @@
+package com.juul.tuulbox.test
+
+import kotlinx.datetime.Clock
+import kotlinx.datetime.Instant
+import kotlinx.datetime.LocalDateTime
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.toInstant
+
+private val defaultTime: Instant = LocalDateTime(
+    year = 2022,
+    monthNumber = 3,
+    dayOfMonth = 8,
+    hour = 8,
+    minute = 15,
+).toInstant(TimeZone.UTC)
+
+public class ManualClock(
+    private var time: Instant = defaultTime
+) : Clock {
```

Comment:
IMHO, we should drop the default time, unless there is a standard for this default?

```suggestion
public class ManualClock(
    private var time: Instant,
) : Clock {
```

_Library consumers can "fake" a default constructor, if desired:_

```kotlin
fun ManualClock() = ManualClock(defaultTime)
```